auto_tuning/proxy/src/proxy-runners/requirements.txt

```diff
@@ -0,0 +1,2 @@
+matplotlib
```

Comment:
Why do we need matplotlib?

Reply:
I added an optional output from matplotlib